config/jail.conf

```diff
@@ -745,6 +745,37 @@ banaction = %(banaction_allports)s
 bantime  = 604800  ; 1 week
 findtime = 86400   ; 1 day
 
+# Blacklist Jail suggested by mitchellkrogza
+# I based this on the recidive filter but this logs bad IPs to a text file ip.blacklist
+# This ensures long banned IPs are persistent across reboots and restarts
+# Requires /etc/fail2ban/action.d/blacklist.conf
+# Requires /etc/fail2ban/filter.d/blacklist.conf
+# Requires /etc/fail2ban/ip.blacklist
+# WARNING: should not be run at the same time as recidive this is meant to better/replace
+# the recidive jail so only one enabled at a time otherwise duplicates will occur.
+# I recommend Fail2Ban log level should be set to INFO
+# The action file is well tested, well documented and cleans up after itself 
+# Unlike other persistent ban filters out there this one cleans up properly when a ban
+# has expired. The ban is removed from IPTables and also from the ip.blacklist file.
+# I have also wildcarded the logfile location of Fail2ban to ensure even compressed
+# .gzip logs are included and log rotation is set to 13 months to ensure a full year
+# of log files are referenced.
+# This jail could either become its own jail or could at some point replace or enhance
+# the existing recidive jail?
+#
+# I am very new to GITHUB so I hope I am doing this the right way. 
+# Please Dont punish me if I am not. Just trying to contribute to the greatest
+# Security Tool for Ubuntu and Linux !!! 
```

Comment:
It is rather a strange replacement for recidive jail, but the "persistence" of bans is normally guaranteed since 0.9 also, at least if sqlite-db used...